### ISSUE TITLE
I'd highly recommend to use Makefile instead so we can even add this to our CI.

### ISSUE THREAD

**Issue by disa6302** [aws-samples/amazon-kinesis-video-streams-demos#106]
I'd highly recommend to use Makefile instead so we can even add this to our CI. 

To add this to the CI, you just need to create a Makefile that has "test" as the target, and add the directory path to the target in .travis.yaml below:

https://github.com/aws-samples/amazon-kinesis-video-streams-demos/blob/e246e81370751088cb82b32be6e5176bb413d0f7/.travis.yml#L17

It's definitely not the best test, but it can add some sanity check at least.

_Originally posted by @lherman-cs in https://github.com/aws-samples/amazon-kinesis-video-streams-demos/pull/64#discussion_r505072363_

**Comment by disa6302** [comment]
The makefile works on systems. However, travis has a log limit https://docs.travis-ci.com/user/common-build-problems/#log-length-exceeded, which causes a problem since `mvn package` generates a lot of logs. This error can be solved using `mvn package -q` which will set verbosity to error. However, due to no logs for prolonged period, travis build fails with the following error: `Build times out because no output was received `.

We need to look into the best way to add the consumer-java to .travis.yml.

**Comment by lherman-cs** [comment]
It's going to be very silly to do this, but maybe we can add something like,

```sh
while 1; do echo "Processing" && sleep 1; done &
```

**Comment by disa6302** [comment]
Yes. I was thinking something along the same lines. But thought of this as a last ditch effort after exhausting all other options :/
